Roles/Neutral/PlagueBearer.cs

```diff
@@ -71,13 +86,29 @@ public static void SendRPC(PlayerControl player, PlayerControl target)
         writer.Write(target.PlayerId);
         AmongUsClient.Instance.FinishRpcImmediately(writer);
     }
+    public override void ApplyGameOptions(IGameOptions opt, byte playerId)
+    {
+        if(PestilenceList.Contains(playerId))
+            opt.SetVision(PlagueBearer.PestilenceHasImpostorVision.GetBool());
+    }
+    public override bool OnRoleGuess(bool isUI, PlayerControl target, PlayerControl pc, CustomRoles role)
```

Comment:
I forgot to add a `, ref bool GuesserSuicides `  here (and sync all other OnRoleGuess methods.) instead of this wack get; set;

Reply:
Yeah, that's exactly what I wanted to do